fixtures/app/extensions/customer-accounts-ui/src/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import {render, Banner} from '@shopify/customer-account-ui-extensions-react';
+import en from "../locales/en.default.json";
```

Comment:
should we not inject the translations into the App function instead?
similar to [checkout ui extensions](https://github.com/Shopify/cli/blob/main/fixtures/app/extensions/checkout-ui/src/index.jsx)

Reply:
Yes exactly, this will be the future state. The problem is that this can only be done after the following two tickets have been implemented:

- https://github.com/Shopify/core-issues/issues/43320
- https://github.com/Shopify/core-issues/issues/43323

The thing is, in order to do the other two tickets, this PR needs to be merged first.